Utils/azure-explorer-common/src/com/microsoft/tooling/msservices/serviceexplorer/azure/arm/deployments/DeploymentNodePresenter.java

```diff
@@ -40,11 +42,12 @@
 
 public class DeploymentNodePresenter<V extends DeploymentNodeView> extends MvpPresenter<V> {
 
-    public void onGetExportTemplateRes(String template, File file) {
+    public void onGetExportTemplateRes(String template, File templateFile, String parameter, File parameterFile) {
```

Comment:
What should onGetExportTemplateRes explain?
is it related with on? get(why write file) ?

Reply:
change it to `onExportTemplateRed`, methods in Presenter will be triggered by correspond method in View